java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6207_java.html

```diff
@@ -0,0 +1,48 @@
+<p>In Java 16 records represent a brief notation for immutable data structures. Records have autogenerated implementations for constructors with all
+parameters, <code>getters</code>, <code>equals</code>, <code>hashcode</code> and <code>toString</code>. Although these methods can&nbsp;still be
```

Comment:
there is a suspicious character `&nbsp;`  here, as well as on almost every example of code!